Marlin/temperature.cpp

```diff
@@ -512,24 +513,27 @@ inline void _temp_error(int e, const char *serial_msg, const char *lcd_msg) {
 
 void max_temp_error(uint8_t e) {
 // Temp error has been reset
-  if (Cartridge__GetPresentCheck()){
+  if (Cartridge__GetPresentCheck()) {
     if (time_since_last_err[e] == 0) {
       time_since_last_err[e] = millis();
     }
   // There has been a recent error, if was more than a second ago, it is probably an error
     else if (millis() > time_since_last_err[e] + TEMP_ERROR_INTERVAL) {
-      if(!Cartridge__FFFNotPresentHysteresis()) 
+      if(Cartridge__Present(e)) 
```

Comment:
So the point of the Cartridge__FFFNotPresentHysteresis function isn't to provide hysteresis for the error, it's to provide a time delay after the cartridge is inserted before triggering the delay. If this isn't done, the error will be checked before the first temperature reading is taken from the cartridge. The temp will be reported as 1100, and the protection will be tripped. I would test this extensively before removing Cartridge__FFFNotPresentHysteresis, as I don't think this will work to prevent that scenario.